### ISSUE TITLE
How to fine-tune FGT on my own dataset

### ISSUE THREAD

**Issue by CSU-NXY** [hitachinsk/FGT#28]
Hi, thanks for your excellent project. I'd like to fine-tune FGT on my own video frames, but don't know how to do it. Can you give me some hint?

**Comment by hitachinsk** [comment]
There are two common ways for this goal. First, you can retrain the FGT pretrained model on your own dataset with a lower learning rate (maybe 1e-5). Second, you can distill the information from pretrained FGT model to your own model with specifically designed supervision. Hope these tips help you.

**Comment by CSU-NXY** [comment]
Thank you!
